### ISSUE TITLE
Inline create a new variable

### ISSUE THREAD

**Issue by alicolville** [alicolville/shortcode-variables#27]
On "List all shortcodes" page, rather than have to click away each time a new page, allow editors to add shortcodes inline / ajax.

Detail from request:

a) Open just a new line and let user insert values directly and allow the already existing "Save" button to work

b) Add a new command button next to the "Save" button with title and function "Go to Editor" and then open the editor window

The advantage is that if the user has just text or plain HTML to insert then the work is done in lightspeed

**Comment by alicolville** [comment]
Going out in 4.0
